Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -283,11 +283,19 @@ class PlaygroundController: ObservableObject {
     var customerSessionClientSecret: String?
     var paymentMethodTypes: [String]?
     var amount: Int?
-    var checkoutEndpoint: String = PaymentSheetTestPlaygroundSettings.defaultCheckoutEndpoint
     var addressViewController: AddressViewController?
     var appearance = PaymentSheet.Appearance.default
     var currentDataTask: URLSessionDataTask?
 
+    var checkoutEndpoint: String {
+        get {
+            settings.checkoutEndpoint
```

Comment:
Should this be `settings.checkoutEndpoint ?? PaymentSheetTestPlaygroundSettings.defaultCheckoutEndpoint`?